lib/logstash/bundler.rb

```diff
@@ -0,0 +1,125 @@
+require "bundler"
```

Comment:
ok, sure, I buy that. let's open a separate issue for that ok? then we can cleanup all monkey patches in one PR?